modular_zubbers/code/datums/components/vore/_defines.dm

```diff
@@ -9,27 +9,27 @@
 /// Value used for belly versioning independent of savefile version
 #define VORE_BELLY_VERSION "1.0"
 /// Maximum amount of bellies allowed within one layout
-#define MAX_BELLIES 10
+#define MAX_BELLIES CONFIG_GET(number/vore_max_bellies)
```

Comment:
```suggestion
#define MAX_BELLIES CONFIG_GET(number/vore_max_bellies) // Default: 10
```
Add some code comments that tell the default for future readability on all of these too.